src/test/java/com/kodilla/ecommercee/services/SimpleEmailServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.ecommercee.services;
+
+import com.kodilla.ecommercee.domains.Mail;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Brakuje @SpringBootTest, to może powodować ze musiałeś dodać adnotacje w klasie aplikacji

Reply:
Nie ma to związku, sprawdziłem ponownie u siebie i bez ComponentScan i dodaniu SpringBooTest wystepuje blad o ktorym pisalem na poczatku